docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
+| Regular security testing can increase stakeholder trust in TREs                                                                             | Knowledge that regular security testing occurs will help to ensure stakeholders, including researchers and data providers, can trust that the data they work with or are responsible for is secure within a TRE.                                                                                                                                                                                                                                  |
 
-### 4.2 Security testing
+### 4.2 Encryption
 
-The ability of the organisation to gain assurance in the security of an IT system by testing or attempting to breach some or all of that system's security.
+The ability of the organisation to deploy and manage encryption to protect information assets, including data for TRE research projects.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement                                             | Guidance                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TREs can encrypt data as an extra layer of protection | Encryption can be seen as an extra layer of protection for sensitive data already protected by the security controls of a TRE. This prevents unauthorised access to the data even if the storage media is compromised. Approved researchers can be granted access to the mechanism to decrpyt the data before using it in their research.                                                                         |
+| TREs should encrypt data when in transit              | TRE architectures that are built such that data gets transmitted over networks (between machines), should ensure data is encrypted in transit. This safeguards the data from interception or tampering during transmission.                                                                                                                                                                                       |
```

Comment:
It might help to split this into internal and external,

- Internal – between machines within a TRE (should)
- External – between the TRE and machines outside of the TRE (must) _e.g._ remote desktop sessions, data ingress, data egress.